knowledge_repo/converters/ipynb.py

```diff
@@ -1,11 +1,23 @@
 import os
+import nbconvert
 
 from ..converter import KnowledgePostConverter
 from .._version import __optional_dependencies__
 
 TEMPLATE = '''
 {%- extends 'markdown.tpl' -%}
 
+{%- block data_javascript scoped %}
+{% set div_id = uuid4() %}
+<div id="{{ div_id }}"></div>
+<div class="output_subarea output_javascript {{ extra_class }}">
+<script type="text/javascript">
```

Comment:
Is this `element` variable expected to be present by notebook javascript cells? Or is this simply for debug purposes?